ca/certificates.go

```diff
@@ -173,7 +168,6 @@ func (rca *RootCA) RequestAndSaveNewCertificates(ctx context.Context, paths Cert
 		if err == nil {
 			break
 		}
-		log.Warningf("error fetching signed node certificate: %v", err)
```

Comment:
Is it being removed because we return it to something that logs it a second time?